app/views/_config-file-params.html

```diff
@@ -1,12 +1,26 @@
-<div ng-repeat="(serverName, data) in secretData" class="image-source-item">
+<div ng-repeat="(serverName, data) in secretData.auths" class="image-source-item">
   <h3>{{serverName}}</h3>
-  <dt>username</dt>
-  <dd class="word-break">{{data.username}}</dd>
-  <dt>password</dt>
-  <dd ng-if="view.showSecret">
-    <copy-to-clipboard clipboard-text="data.password" display-wide="true"></copy-to-clipboard>
-  </dd>
-  <dd ng-if="!view.showSecret">*****</dd>
-  <dt>email</dt>
-  <dd class="word-break">{{data.email}}</dd>
+  <div ng-if="data.username || data.password || data.email">
```

Comment:
wondering if this is necessary/helpful, nothing will show if none exist anyhow.

Reply:
It looks a little odd to me because the header is underlined with nothing below. Opinion?

![openshift web console 2017-09-12 15-00-05](https://user-images.githubusercontent.com/1167259/30343316-17aac22a-97cb-11e7-97b7-24f210c8645c.png)

Reply:
Agreed. I would leave it.

Reply:
Oh, sorry, I was thinking of the div below when the data is missing. Yeah this div isn't needed, will fix